Release of API version 1.0.0-rc.1 (#74)
camaraproject/OTPValidation: README.md

## Meetings

Current schedule, registration, & meeting links are available on the confluence page: [Meetings information](https://wiki.camaraproject.org/display/CAM/OTPValidation)
  
## Status and released versions

* Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.

> * **The latest available version of CAMARA OTPValidaton API is 1.0.0-rc.1 This is the release candidate of the first stable version candidate of the API**. The pre-release is suitable for implementors.

Reviewer: ```suggestion
* **The latest pre-release of OTPValidaton is r1.1. The pre-release contains the release candidate of the first stable version of the one-time-password-sms API**. The pre-release is suitable for implementors.
```

> * **The latest available release and version of CAMARA OTPValidaton API is 1.0.0-rc.1 This is the first stable version candidate of the API**. It is suitable for implementors and could be use with customers in productive environments.

Reviewer: I don't that the release candidate should be used already in productive environment - it is a pre-release. Therefore also "latest" is not the right term (for GitHub this is a public release). And to be consistent with the changelog just a proposal:

```suggestion
* **The latest available version of CAMARA OTPValidaton API is 1.0.0-rc.1 This is the release candidate of the first stable version candidate of the API**. The pre-release is suitable for implementors.
```
